golang/x/relay/client/cli/tx.go

```diff
@@ -108,3 +149,55 @@ func GetCmdNewRequest(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// GetCmdMarkNewHeaviest creates a CLI command to update best known digest and LCA
+func GetCmdMarkNewHeaviest(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "marknewheaviest <ancestor> <currentBest> <newBest> [limit]",
+		Short: "Updates best known digest and LCA",
+		Long:  "Updates best known digest and LCA. Ancestor, current best, and new best are hex.  Limit is an integer.",
```

Comment:
extra space here. `are hex.  Limit`

We should consider using line breaks in this message.